blog/2024/11-30-android-tv-18/index.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Jellyfin for Android TV 0.18
+authors:
+  - nielsvanvelzen
+slug: androidtv-v0.18.0
+tags: [release, android-tv]
+---
+
+Version 0.18 of our Jellyfin for Android TV app is ready! Exciting features like lyrics and media segments are now available, along with
+significant improvements to subtitles and more.
+
+{/* truncate */}
+
+This release marks another milestone in our journey to delivering the best media app on Android TV. Jellyfin 10.10 or newer is required. A
+complete list of changes, including their respective pull requests, can be found on
+[GitHub](https://github.com/jellyfin/jellyfin-androidtv/releases/tag/v0.18.0). I've listed a subset of all changes below.
+
+### Skipping intros, outros and more with media segments
+
+Media segments are a new feature added in Jellyfin 10.10. A segment provides metadata for specific parts of a video or audio file. This can
+be used to mark an intro in your episode or the credits of a movie. The Android TV app can now use those segments to skip parts. By default,
+the app will ask to skip intros and outros, but you can fine-tune this to your liking within the app preferences. All media segment types
+are supported: intro, outro, preview, recap and commercial with two actions to choose from:
+
+- **Skip**
```

Comment:
Is none or do nothing an option like on web?

Reply:
I do. Only mentioning there being 2 options makes it sound like you must have it either auto-skip or display the button. But you _can_ choose to simply ignore the segment if you like.

Reply:
Added

> - **Do nothing**  
>   The segment will be ignored, with no skips or popups.